app/controllers/tickets_controller.rb

```diff
@@ -28,6 +29,16 @@ def set_client
     @session = params[:session]
   end
 
+  def check_email(email)
+    regex = /\S+@\S+[\.][0-9a-z]+/
+    stripped = strip_tags(email)
+    test = regex.match stripped
+    unless test
+      response.status = 401
```

Comment:
does it mean that if `test` fails then the request is not `authorized`? just checking if it is the right response